### PR TITLE
feat: Improve UI of EventDetailsFragment

### LINKED ISSUE #221: Make the UI of Event Details Fragment comparable to EventBrite’s details page
Working on it :+1:

### PR DESCRIPTION
**Pull request by dreadpool2**
Fixes #221 #227 
*Added Organizer name and logo in Organizer section
*Resolved invisible attendee fragment issue
*Improved UI of EventDetailsFragment

### CODE REVIEW

**Comment by dreadpool2** [comment]
Screenshots after update

screenshot_20180630-125926
screenshot_20180630-125934
screenshot_20180630-125943
screenshot_20180630-125948

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -7,4 +7,9 @@
     <color name="black">#000000</color>
     <color name="grey">#ece9e9</color>
     <color name="greyMore">#918e8e</color>
+
+    <color name="darkGrey">#333333</color>
+    <color name="lightGrey">#545454</color>
+    <color name="lighterGrey">#757575</color>
```
Comment (iamareebjamal) [inline app/src/main/res/values/colors.xml]: snake case

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -7,9 +7,10 @@
     <!--event details-->
     <string name="event_card_date">Event Date :</string>
     <string name="report_event">Report Event</string>
-    <string name="about_event">About Event</string>
+    <string name="about_event">About</string>
```
Comment (iamareebjamal) [inline app/src/main/res/values/strings.xml]: about

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -7,9 +7,10 @@
...
     <string name="see_more">See more</string>
-    <string name="about_organizer">About Organizer</string>
+    <string name="event_host_details">Event host details</string>
+    <string name="about_organizer">Organizer</string>
```
Comment (iamareebjamal) [inline app/src/main/res/values/strings.xml]: organizer

**Comment by dreadpool2** [comment]
PR Updated :+1:

**Pull request merged**